test/suites/osconfig/systemd-resolved.robot

```diff
@@ -170,3 +164,12 @@ Restore Systemd-Resolved
         ...    sudo=True    return_rc=True    return_stdout=True    return_stderr=True
         Should Be Equal As Integers    0    ${rc}
     END
+
+Check ResolveConf Option Presence
+    [Documentation]    Check if the 'resolvConf' option is present in the kubelet
+    ...    configuration file. Return a none-zero code if not present.
+
+    ${rc}=    Execute Command
+    ...    grep -qE "^resolvConf:.*${RESOLVE_CONF_FILE}" ${KUBELET_CONFIG_FILE}
+    ...    sudo=True    return_rc=True    return_stdout=False    return_stderr=False
+    RETURN    ${rc}
```

Comment:
Can we include here
```
Should Not Be Equal As Integers 0 ${rc}
```

All usages of these keyword do that.

+ rename to "ResolvConf Should Be Present" or smth like this

Reply:
I think this one does not, right?
https://github.com/openshift/microshift/pull/2667/files#diff-ec352c7ad6712e0a73a467e27ee66dc0b1ae368db3453136387b8da7a5406266R33

Reply:
Ah, obviously... We could then have an argument to expect existence or lack of existence, but that's your choice then